lib/mix/tasks/surface/convert_syntax.ex

```diff
@@ -0,0 +1,211 @@
+defmodule Mix.Tasks.Surface.ConvertSyntax do
+  @shortdoc "Converts .sface files and ~H sigils from pre-v0.5 to v0.5 syntax"
+
+  @moduledoc """
+  Converts .sface files and ~H sigils from pre-v0.5 to v0.5 syntax.
+
+      mix surface.convert_syntax "lib/**/*.{ex,exs}" "test/**/*.{ex,exs}"
```

Comment:
On my computer, if I run `mix surface.convert_syntax`, the `.sface` files are not converted. I have to force it with a file pattern `mix surface.convert_syntax "lib/**/*.sface"`. I believe we should handle sface files by default.

Reply:
The converter only runs against paths configured in `.formatter`.